ci/taos/checker-pr-audit-async.sh

```diff
@@ -103,7 +103,7 @@ export dir_worker=$dir_worker
 # check if dir_commit folder exists, then, create dir_commit folder
 # let's keep the existing result although the same target directory already exists.
 cd $dir_ci
-export dir_commit=${dir_worker}/${input_date}-${input_pr}-${input_commit}
+export dir_commit=${dir_worker}/${input_pr}-${input_date}-${input_commit}
```

Comment:
Which better?
1) display the folders with prefix "pr_date" to monitor the number of PRs per a day. (Currently)
2) display the folders with prefix "pr_no" to find repo folders of a PR easily.

Reply:
@ohsewon  Please append the below statement in the commit body (=commit description) for history. 

`Fixed issue #410.`

Reply:
updated.